helm/sematic-server/templates/cleaner-cron.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: batch/v1
+kind: CronJob
```

Comment:
Please allow the entire job to be disabled on deployment by enclosing the entire contents in an if based in whether the `values.yaml` contains an entry for `cleaner`.

Reply:
I don't think we should allow it to be disabled unless we have a customer request to do so.

Reply:
it should be fine to _allow_ disabling it, and keep the default as enabled.  especially useful in the short term in case there are any issues that need to get ironed out and customers want to disable it until the next release.

Reply:
I just tend to err on the side of "reduce the configuration surface area as much as possible" since we can't easily test exotic configuration combinations.

Reply:
made configurable via a bool after all